src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds

```diff
@@ -37,5 +37,24 @@
     </Project>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(__BuildType)' == 'Release'">
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroName)' == 'rhel'" Include="rhel/Microsoft.NETCore.Runtime.CoreCLR-symbols.pkgproj">
```

Comment:
For the nupkg specifically the convention is `.symbols.nupkg`. At this level `.Symbols` seems like a decent way to indicate that in a project name.